hikari/internal/fast_protocol.py

```diff
@@ -32,7 +32,7 @@
     from typing_extensions import Self
 
 _Protocol: FastProtocolChecking = NotImplemented
-_IGNORED_ATTRS = typing.EXCLUDED_ATTRIBUTES + ["__qualname__", "__slots__"]
+_IGNORED_ATTRS = frozenset(tuple(typing.EXCLUDED_ATTRIBUTES) + ("__qualname__", "__slots__"))
```

Comment:
```suggestion
_IGNORED_ATTRS = frozenset(typing.EXCLUDED_ATTRIBUTES) | {"__qualname__", "__slots__"}
```

Reply:
That could work too, the main reason for the tuple was a little simplificacion, as I thought that `|` a frozenset would yield a normal set, but it's apparently a frozen one too

```py
>>> a = frozenset({1,2,3}) | {4,5,6}
>>> a
frozenset({1, 2, 3, 4, 5, 6})
```

TIL!